spec/requests/locations/post_spec.rb

```diff
@@ -10,10 +10,31 @@ module Mutations
             post '/graphql', params: { query: query() }
           end.to change { Location.count }.by(1)                       
         end
+
+        it 'returns all locations' do 
+          post '/graphql', params: { query: query }
+          post '/graphql', params: { query: query }
```

Comment:
should we be calling this twice?

Reply:
The intention behind calling it twice is to ensure that if multiple locations are in the database it will return all of them.